tools/mo/openvino/tools/mo/middle/fusings.py

```diff
@@ -46,9 +46,6 @@ def find_and_replace_pattern(self, graph: Graph):
         for_graph_and_each_sub_graph_recursively(graph, fuse_pad)
         for_graph_and_each_sub_graph_recursively(graph, lambda G: G.clean_up())
 
-        # Mark nodes with attr 'can_be_fused': False to disable fusing for specified nodes
-        for_graph_and_each_sub_graph_recursively(graph, lambda graph: mark_unfused_nodes(graph, argv.finegrain_fusing))
-
```

Comment:
why did we remove it?
@jane-intel, could you please comment on this change if it is correct?

Reply:
because this transformation is run only if `finegrain_fusing` is enabled, but since we removed this argument this marking pass is newer run I removed it

Reply:
What is the default value of this option?

Reply:
Remained question. Please double-check if this is correct change. Don't we skip some fusing transformation this way?

Reply:
I double checked. Default value is `None` which means skip and do not mark anything which is equivalent of not running this pass 
![image](https://user-images.githubusercontent.com/5703039/212139896-db96b8f7-1d18-411a-9c4f-c48207fe512a.png)